pkg/agent/client.go

```diff
@@ -501,6 +501,18 @@ func (a *Client) Serve() {
 			ctx, ok := a.connManager.Get(data.ConnectID)
 			if ok {
 				ctx.send(data.Data)
+			} else {
+				klog.V(2).InfoS("received DATA for unrecognized connection", "connectionID", data.ConnectID)
+				a.Send(&client.Packet{
```

Comment:
Again no point is sending a close on connectID == 0.

Reply:
Same comment as above.